sources/packages/api/src/utilities/application-utils.ts

```diff
@@ -47,3 +55,16 @@ export const transformToApplicationSummaryDTO = (
     status: application.applicationStatus,
   } as ApplicationSummaryDTO;
 };
+
+/**
+ * Throw validation exception on Application date overlap or PIR Pending.
+ */
+export const throwExceptionForPIRDateOverlap = (
+  application: Application | SFASApplication | SFASPartTimeApplications,
```

Comment:
It could be simplified as a Boolean variable also.

Reply:
With updated code the util became obsolete and removed.